S/StatsPlots/Compat.toml

```diff
@@ -17,5 +17,12 @@ julia = "0.7-1"
 ["0-0.10.1"]
 Widgets = "0.4-0"
 
-["0.10.2-0"]
+["0.10.2-0.10"]
 Widgets = "0.5-0"
+
+["0.11-0"]
+Observables = "0.2.2-0.2"
+Plots = ["0.14", "0.25"]
```

Comment:
This is probably not what you meant; what about all Plots versions in between?

Reply:
Thank you, it definitely is not. @kescobo how did you generate the Project.toml file?

Reply:
Manually :-/.

The old require file had 0.14, I thought this was an interval 😳

Reply:
No, you have to list every breaking version you are compatible with
```compat
Plots = "0.14, 0.15, 0.16, 0.17, 0.18, ..."
```

Reply:
@fredrikekre @mkborregaard So what's the best way to handle this? Is there a way to make that edit associated with this PR, or should we cancel this PR and start a new one with that fix?

Reply:
Just edit and then retrigger the bot and this PR will be updated.

Reply:
bump

Reply:
I'm fine with just saying "0.25" and getting it over with. I don't think you would need the newest StatsPlots version if you are on an older Plots version. It'll require editing often, as minor versions come out often on Plots.
IMHO this would be a good reason to push out Plots 1.0.0 anyway.

Reply:
@mkborregaard did you want me to do another PR to edit?

Reply:
oh that would be great :-)